src/VolleyManagement.Backend/VolleyManagement.UI/Areas/Mvc/ViewModels/GameResults/GameResultViewModel.cs

```diff
@@ -81,6 +81,17 @@ public GameResultViewModel()
         /// </summary>
         public byte GameNumber { get; set; }
 
+        /// <summary>
+        /// Gets display number of the game in the tournament
+        /// </summary>
+        public string DisplayGameNumber
```

Comment:
You can use Expression-Bodied Properties

Reply:
ok